data/korath/korath hails.txt

```diff
@@ -0,0 +1,1127 @@
+		"arepas Drak ilrotrinef akis."          # because [the] Drak protect us
+		"ralraw iacfam ipekafikra ipihsalref."  # lest antimatter [from an Archon] end [you]
+		"sokro ipatrihlref ramanabfec Quarg."   # to avoid a Quarg's vengeance
+
+phrase "exile: I do bad things to my enemies"
+	phrase
+		"exile: I perform harmful acts upon"
+	word
+		" "
+	phrase
+		"exile: people who offend, attack, or oppress me"
+	word
+		.
+		.
+		!
+
+phrase "efreti: I do bad things to my enemies"
+	phrase
+		"efreti: I perform harmful acts upon"
+	word
+		" "
+	phrase
+		"efreti: people who offend, attack, or oppress me"
+	word
+		.
+		.
+		!
+
+phrase "exile: I perform harmful acts upon"
+	word
+		"Bapamef ranres alihfen"            # [I] joyfully devour
+		"Ratto'afis et famas ataler"        # [I] subsume into an inferno
+		"Raiakrahfeb abrak robfa"           # [I] mercilessly slaughter
+		"Ratotamef railak raikafet"         # [I] perform the dance of death {in relation to | with the subject being}
+		"Li'imalkeref rafadra ilas"         # I neutralize the threat of
+		"Rarataf ranril asab kimot"         # [I] snack on the skin of
+		"Aga' ikaftiref 'kisap ilas"        # I enjoy the death of
+		"Tisisref imahfet ranres 'alet"     # [I] violently reeducate
+		"Ratilehfef akrid ralet atakebamaf" # [I] give love-taps of doom [to]
+		"Tikimennref alade' tikbimatuba"    # [I] apocalyptically tickle
+
+phrase "efreti: I perform harmful acts upon"
+	word
+		"Ratotamef railak raikafet"         # [I] perform the dance of death {in relation to | with the subject being}
+		"Li'imalkeref rafadra ilas"         # I neutralize the threat of
+		"Rarataf ranril asab kimot"         # [I] snack on the skin of
+		"Osa rahimas atac bimar"            # I pity the fate of
+		"Adam ikafsiref mkitah ipat"        # I enjoy the death of
+		"Rassowrok raparebes pihsapek ataces" # [I] show [the] final truth to
+		"Sititref inabfes ralret mapes"     # [I] violently reeducate
+		"Rasipebfef akriv raps asakecanaf" # [I] give love-taps of doom [to]
+		"Kahafef lratro-lratro lranok ipat" # [I] sculpt bone statues out of
+		"Adam satik asotpeb"                # I do not mourn
+
+phrase "exile: people who offend, attack, or oppress me"
+	word
+		"rahlut rapalafet aga' nrag lareh"  # victims of my righteous rage
+		"atelef nrag ohlegref kinram"       # those who invade {our|the} {sky | star system}
+		"rimavlek ranres otraa'a'okobet"    # [people who are] entwined with my despair
+		"pavavreb asi'uren akit"            # our genocidal invaders
+		"atelef nrag rata pavavref akit"    # those who would colonize {us|our home}
+		"rafadra inah bosip aga'"           # [people who are] threats to my life
+		"aofe' nralu pusuh abrak kimot"     # all skinless fools [who threaten us]
+		"kapavreb ranres ikap 'e"           # villains with hearts of ice
+		"Humanika nrag tasik jikalebuut"    # uncooperative humans
+		"kagaf bosip is Nraol Alah"         # {walking corpses|helpless victims} in Ember Space
```

Comment:
I saw much worse things than this come out of the cipher. There were many words I had to manually correct to avoid obscene or otherwise objectionable content.